timesketch/lib/analyzers/yetiindicators.py

```diff
@@ -131,26 +202,69 @@ def mark_event(
             event: a Timesketch sketch Event object.
             neighbors: a list of Yeti entities related to the indicator.
         """
-        event.add_emojis([emojis.get_emoji("SKULL")])
-        tags = []
+        tags = {slugify(tag) for tag in indicator["relevant_tags"]}
         for n in neighbors:
```

Comment:
Nit: I prefer to spell it out:
```suggestion
        for neighbour in neighbours:
```